plugins/woocommerce/client/legacy/css/_variables.scss

```diff
@@ -32,7 +32,11 @@ $subtext:           #767676 !default;                                  // small,
 	--wc-secondary: #{$secondary};
 	--wc-secondary-text: #{$secondarytext};
 	--wc-highlight: #{$highlight};
-	--wc-highligh-text: #{$highlightext};
+	--wc-highlight-text: #{$highlightext};
```

Comment:
what was the reason for renaming this? i mean it's obviously spelled incorrectly but so is the variable it references? I couldn't find usage of this variable otherwise in your code

Reply:
Just bugged me; I'll revert so its easier to review.